packages/appium/test/unit/extension/extension-config.spec.js

```diff
@@ -300,11 +300,8 @@ describe('ExtensionConfig', function () {
       });
 
       describe('when the extension is not actually installed', function () {
-        it('should throw', function () {
-          expect(() => config.require('fake')).to.throw(
-            ReferenceError,
-            /^could not find a driver installed at .+flotsam/i
-          );
+        it('should throw', async function () {
+          await config.requireAsync('fake').should.be.rejected;
```

Comment:
Is it possible to use `rejectedWith(/<error message>/)` in both tests to distinguish the error messages? Otherwise both of the changed tests are identical.

Reply:
updated